k8s/overlays/main_versions/prod/kustomization.yaml

```diff
@@ -33,52 +36,55 @@ images:
     newTag: 0.15.0
   - name: gridcapa-rao-runner
     newName: docker.io/farao/gridcapa-rao-runner-with-xpress
-    newTag: v1.15.0
+    newTag: v1.16.2
   - name: gridcapa-app
     newName: docker.io/farao/gridcapa-app
-    newTag: v1.10.0
+    newTag: v1.9.0
```

Comment:
We need to check that the current configmap is compatible with the previous version of gridcapa-app

Reply:
here are you talking about a specific configmap ? app meta data  configmap...